src/app.router.js

```diff
@@ -21,7 +22,7 @@ import PasswordChangeSuccessDialog from './account/PasswordChangeSuccessDialog';
 
 function WrAppadApp(props) {
```

Comment:
Not your change, but I just wanted to point it out. I had to do a double take when I first read it 😅